Makefile

```diff
@@ -90,10 +90,12 @@ PULL_POLICY ?= Always
 LDFLAGS := $(shell source ./hack/version.sh; version::ldflags)
 
 # 'functional tests' as the ginkgo filter will run ALL tests ~ 2 hours @ 3 node concurrency.
-E2E_UNMANAGED_FOCUS ?= "functional tests - unmanaged"
+E2E_UNMANAGED_FOCUS ?= "Cluster API E2E tests"
```

Comment:
This is essentially disabling the functional tests in favour of the E2E tests?

Reply:
Actually I was going to run these tests by setting the focus in test-infra for both AWS specific periodic e2e and CAPI e2e.
But you are right, I will revert it to normal, then only enable CAPI e2e in the new periodic infra test.

Reply:
Ok, didn't see this gap analysis. Taking a look.